src/In.ProjectEKA.HipService/Link/LinkPatientController.cs

```diff
@@ -64,11 +64,11 @@ private ActionResult ResponseFrom(ErrorRepresentation errorResponse)
         {
             return errorResponse.Error.Code switch
             {
-                ErrorCode.OtpExpired => (ActionResult) BadRequest(errorResponse),
+                ErrorCode.OtpExpired => (ActionResult) Unauthorized(errorResponse),
                 ErrorCode.MultiplePatientsFound => NotFound(errorResponse),
                 ErrorCode.NoPatientFound => NotFound(errorResponse),
                 ErrorCode.OtpGenerationFailed => StatusCode(StatusCodes.Status500InternalServerError, errorResponse),
-                ErrorCode.OtpInValid => NotFound(errorResponse),
+                ErrorCode.OtpInValid => BadRequest(errorResponse),
```

Comment:
Both cases it's unauthorized only no? at least in the swagger.

Reply:
Swagger says 400 and 401 both for invalid otp at least for link api. Not really sure what we need to fallow.

Reply:
https://stackoverflow.com/questions/30078459/what-should-be-http-status-code-for-credentials-expired-error-exception